api/query/post.ts

```diff
@@ -48,11 +48,7 @@ const logic = async ({
   if (targetConditions.length === 0) {
     return queryRoam({
       authorization,
-      body: {
-        conditions: body.conditions,
-        returnNode: body.returnNode,
-        selections: body.selections,
-      },
+      body: { ...body }, // TODO type fix
```

Comment:
I was working on a version before https://github.com/samepage-network/roam-samepage/commit/49de5ed05e1d23931e2dfc451f39fab47528b827, so this will require some type fixing